specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -348,6 +348,14 @@ model Test {
   @added(APIVersions.v2024_07_01_preview)
   metricsReferenceIdentityId?: string;
 
+  @doc("Type of the managed identity built in load test engines")
+  @added(APIVersions.v2024_12_01_preview)
+  engineBuiltinIdentityType?: ManagedIdentityType = ManagedIdentityType.SystemAssigned;
+
+  @doc("Resource Id of the managed identity built in to load test engines")
+  @added(APIVersions.v2024_12_01_preview)
+  engineBuiltinIdentityId?: string[];
```

Comment:
Ids

Reply:
Update the comment as well to indicate the resource ids of the managed identities to be enabled on the load test engines?

Reply:
done

Reply:
1. Can you clarify on the 'builtin' part. Are there scenarios where we could have non-builtin managed identities as well. Should we have it just and engineIdentityIds (for both the type and id fields?)